frontend/pages/index.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import Head from 'next/head';
+import axios from 'axios';
+
+import { Textarea, Text, Button, Box } from '@chakra-ui/core';
+
+export default function Home() {
+  const [text, setText] = useState('');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [analyzeResult, setAnalyzeResult] = useState(null);
+
+  const handleTextChange = (e) => {
+    setText(e.target.value);
+    setAnalyzeResult(null);
+  };
+
+  const handleOnClick = () => {
+    setIsSubmitting(true);
+
+    // TODO: add the correct endpoint.
+    axios
+      .post('/analyze', {
```

Comment:
^ this is the endpoint

Reply:
Correct me if I'm wrong - there should be 2 endpoints - first is when the page is loaded, second is when the button is clicked which triggers this https://github.com/oren0e/fake-news/blob/bd2387cd4fdfd1a8a9d2c90a0bf8100c4fe89ad9/app.py#L9

You assumed that the first is `/analyze`, but I didn't see any mentions of the second one